<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="modifbtn">
      <property name="text">
       <string>修改标题和图标</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="QFiledialogbtn">
      <property name="text">
       <string>QFIleDialog</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="processdlg">
      <property name="text">
       <string>进度条</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="moverButtion">
        <property name="text">
         <string>移动窗口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="positionButton">
        <property name="text">
         <string>获取位置信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="geometryButton">
        <property name="text">
         <string>修改窗口位置和尺寸信息</string>
        </property>
        <property name="iconSize">
         <size>
          <width>53</width>
          <height>53</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>176</height>
       </size>
      </property>
      <property name="title">
       <string>设置输入框</string>
      </property>
      <widget class="QPushButton" name="inputdlg">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>QInputDialog</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="modelDlg">
      <property name="text">
       <string>显示模块对话框</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="msgbox">
      <property name="text">
       <string>QMessageBox</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>字体选择</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="fontdlg">
         <property name="text">
          <string>QFontDialog</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fontlabel">
         <property name="text">
          <string>颜色变化</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="2" rowspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>颜色选择</string>
      </property>
      <widget class="QPushButton" name="colordlg">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>QColorDialog</string>
       </property>
      </widget>
      <widget class="QLabel" name="colorlabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>351</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="color">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_QT">
    <property name="title">
     <string>学习QT</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="separator"/>
    <addaction name="save_action"/>
   </widget>
   <addaction name="menu_QT"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_action"/>
   <addaction name="save_action"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
  </widget>
  <action name="open_action">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../ico.png</normaloff>../../../../../ico.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="save_action">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../avatar.jpg</normaloff>../../../../../avatar.jpg</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
